Source/Core/PolylineGeometry.js

```diff
@@ -309,9 +309,11 @@ define([
         var k;
 
         var positions = arrayRemoveDuplicates(polylineGeometry._positions, Cartesian3.equalsEpsilon);
-
         var positionsLength = positions.length;
-        if (positionsLength < 2) {
+
+        // A width of a pixel or less is not a valid geometry, but in order to support external data
+        // that may have errors we treat this as an empty geometry.
+        if (positionsLength < 2 || width < 1.0) {
```

Comment:
@bagnell can confirm, but I think less than 1 is fine, it just needs to be greater than 0.